app/components/elements/TextLink.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Box } from 'rebass/styled-components';
+import { TextLink as Text } from '../elements/FontStyles';
+
+export const TextLink = props => {
+  const { linkText, link, ...textLinkProps } = props;
+
+  return (
+    <div>
+      <Box>
+        <Text href={link} {...textLinkProps}>{linkText}</Text>
+      </Box>
+    </div>
+  );
+};
+
+TextLink.propTypes = {
+  label: PropTypes.string.isRequired,
+  link: PropTypes.string,
```

Comment:
Would t would be better to just pass in as `href`, rather than renaming the passed-in prop as it gets applied to the element?